common/include/abstract_protocol.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * abstract_protocol.h
+ *
+ *  Created on: 2020��9��30��
+ *      Author: duanning
+ */
+
+#ifndef INCLUDE_ABSTRACT_PROTOCOL_H_
```

Comment:
Please remove all the path strings in re-include prevention macro definitions, e.g. `INCLUDE_`

Reply:
So, it should be:
#ifndef ABSTRACT_PROTOCOL_H_
?

Reply:
Yes